backends/occa/ceed-occa-okl.c

```diff
@@ -32,7 +32,7 @@ int CeedOklPath_Occa(const Ceed ceed, const char *c_src_file,
   if (!last_dot)
     return CeedError(ceed, 1, "Cannot find file's extension!");
   const size_t okl_path_len = last_dot - okl;
-  strcpy(&okl[okl_path_len],".okl");
+  strncpy(&okl[okl_path_len],".okl",5);
```

Comment:
As a technicality, it is possible to construct paths for which this causes memory corruption

Reply:
Yeah, I'm not super happy it. I put in a note for this to be updated in the OCCA overhaul.